website/templates/project/registrations.mako

```diff
@@ -44,8 +44,8 @@
         %endif
       </div>
       % if 'admin' in user['permissions'] and not disk_saving_mode:
-      <div class="col-md-3">
-        <a id="registerNode" class="btn btn-default" type="button">
+      <div class="col-xs-3 col-sm-4">
+        <a id="registerNode" class="btn btn-success" type="button">
```

Comment:
This line conflicts with ` <a id="registerNode" class="btn btn-default disabled" type="button">` on recent develop. I'm not sure if changing to a `btn-success` is necessary, but it does make it consistent with the `Forks` page. It should have `disabled`, that's removed after MetaSchemas are fetched.